app/resolve/daemon/__init__.py

```diff
@@ -1,2 +1,26 @@
+import requests
+import time
+import logging
+
+import core.config as config
+from core.execute_query import execute_query
+from core.parser import parser
+from core.rquest_dto.result import RquestResult
+from core.settings import TASK_API_BASE_URL
+
+
 def main() -> None:
-    print("hello rabbit world!")
+    logger = logging.getLogger(config.LOGGER_NAME)
+    result = execute_query(parser)
+    if not isinstance(result, RquestResult):
+        raise TypeError("Payload does not match RQuest result schema")
+
+    return_url = f"{TASK_API_BASE_URL}/{result.uuid}/{result.collection_id}"
+
+    for i in range(4):
+        response = requests.post(return_url, data=result.to_dict())
+        if response.status_code == 200:
```

Comment:
On reflection, I think `200` is too specific here.

At minimum `409 Conflict` should not cause further retries. (as it means results have already been received).

For now let's say 2xx and 409 don't retry? (I'm open to not retrying for any 4xx)

Reply:
thanks! pushed the changes